2024-07-05 16:19:05.758 +07:00 [WRN] Failed to determine the https port for redirect.
2024-07-05 16:19:06.290 +07:00 [INF] HTTP GET /api/User/GetAllUser responded 200 in 494.1803 ms
2024-07-05 16:19:08.860 +07:00 [INF] HTTP GET /api/User/GetAllUser responded 200 in 14.9917 ms
2024-07-05 16:19:20.243 +07:00 [INF] HTTP GET /api/User/GetAllUser responded 200 in 43.8020 ms
2024-07-05 16:19:25.494 +07:00 [INF] HTTP GET /api/User/GetAllUser responded 200 in 4.8150 ms
2024-07-05 16:19:31.852 +07:00 [INF] HTTP GET /api/User/GetAllUser responded 200 in 4.0637 ms
2024-07-05 16:19:51.619 +07:00 [ERR] Failed executing DbCommand (8ms) [Parameters=[], CommandType='"Text"', CommandTimeout='30']
SELECT f.userid, f.password, f.username
FROM "Failed" AS f
2024-07-05 16:19:51.636 +07:00 [ERR] An exception occurred while iterating over the results of a query for context type 'OPL_grafana_meilisearch.DataContext'.
Npgsql.PostgresException (0x80004005): 42P01: relation "Failed" does not exist

POSITION: 46
   at Npgsql.Internal.NpgsqlConnector.ReadMessageLong(Boolean async, DataRowLoadingMode dataRowLoadingMode, Boolean readingNotifications, Boolean isReadingPrependedMessage)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at Npgsql.NpgsqlDataReader.NextResult(Boolean async, Boolean isConsuming, CancellationToken cancellationToken)
   at Npgsql.NpgsqlDataReader.NextResult(Boolean async, Boolean isConsuming, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteReader(Boolean async, CommandBehavior behavior, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteReader(Boolean async, CommandBehavior behavior, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteDbDataReaderAsync(CommandBehavior behavior, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)
   at Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.NpgsqlExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
  Exception data:
    Severity: ERROR
    SqlState: 42P01
    MessageText: relation "Failed" does not exist
    Position: 46
    File: parse_relation.c
    Line: 1449
    Routine: parserOpenTable
Npgsql.PostgresException (0x80004005): 42P01: relation "Failed" does not exist

POSITION: 46
   at Npgsql.Internal.NpgsqlConnector.ReadMessageLong(Boolean async, DataRowLoadingMode dataRowLoadingMode, Boolean readingNotifications, Boolean isReadingPrependedMessage)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at Npgsql.NpgsqlDataReader.NextResult(Boolean async, Boolean isConsuming, CancellationToken cancellationToken)
   at Npgsql.NpgsqlDataReader.NextResult(Boolean async, Boolean isConsuming, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteReader(Boolean async, CommandBehavior behavior, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteReader(Boolean async, CommandBehavior behavior, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteDbDataReaderAsync(CommandBehavior behavior, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)
   at Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.NpgsqlExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
  Exception data:
    Severity: ERROR
    SqlState: 42P01
    MessageText: relation "Failed" does not exist
    Position: 46
    File: parse_relation.c
    Line: 1449
    Routine: parserOpenTable
2024-07-05 16:19:51.640 +07:00 [ERR] Error getting All Faileds
Npgsql.PostgresException (0x80004005): 42P01: relation "Failed" does not exist

POSITION: 46
   at OPL_grafana_meilisearch.src.Core.Service.FailedService.GetAllFailedAsync() in /Users/comnarin/Documents/TCC-tech/OPL_grafana_meilisearch/src/Core/Service/FailedService.cs:line 29
   at OPL_grafana_meilisearch.Controllers.FailedController.GetAllFailedAsync() in /Users/comnarin/Documents/TCC-tech/OPL_grafana_meilisearch/Controllers/FailedController.cs:line 28
  Exception data:
    Severity: ERROR
    SqlState: 42P01
    MessageText: relation "Failed" does not exist
    Position: 46
    File: parse_relation.c
    Line: 1449
    Routine: parserOpenTable
2024-07-05 16:19:51.644 +07:00 [INF] HTTP GET /api/Failed/GetAllFailed responded 400 in 51.7390 ms
